### PR TITLE
CONTROLLER: Home

### LINKED ISSUE #20: CONTROLLER: Home

### PR DESCRIPTION
**Pull request by Shrabanac**
Closes #20

### CODE REVIEW

File: JustLearn.Services.Api/Properties/launchSettings.json [hunk 1]
```diff
@@ -1,27 +1,31 @@
...
     "IIS Express": {
       "commandName": "IISExpress",
       "launchBrowser": true,
+      "launchUrl": "swagger",
       "environmentVariables": {
         "ASPNETCORE_ENVIRONMENT": "Development"
       }
     },
-    "JustLearn.Services.Api": {
+    "Random.Services.Api": {
```
Comment (NaynaMallick) [inline JustLearn.Services.Api/Properties/launchSettings.json]: Change Api name Random to JustLearn.

**Review by NaynaMallick — APPROVED** [review]

File: JustLearn.Services.Api/Controllers/HomeController.cs [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+// Copyright (c) MumsWhoCode. All rights reserved.
+// -----------------------------------------------------------------------
+
+using Microsoft.AspNetCore.Mvc;
+using RESTFulSense.Controllers;
+
+namespace JustLearn.Services.Api.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
```
Comment (ShriHumrudha) [inline JustLearn.Services.Api/Controllers/HomeController.cs:10]: Reorder this.

**Review by ShriHumrudha — CHANGES_REQUESTED** [review]
Please add a screenshot

**Comment by Shrabanac** [comment]
image

**Review by NaynaMallick — APPROVED** [review]

**Review by ShriHumrudha — APPROVED** [review]

**Comment by Shrabanac** [comment]
image

**Pull request merged**